demisto_sdk/commands/lint/helpers.py

```diff
@@ -257,58 +259,64 @@ def get_python_version_from_image(image: str) -> float:
         float: Python version X.Y (3.7, 3.6, ..)
     """
     docker_client = docker.from_env()
-    container_obj: Container = None
-    py_num = ""
-    for trial1 in range(2):
+    containers = list()
+    py_num = 2.7
+    # Run two times
+    for _ in range(2):
         try:
             command = "python -c \"import sys; print('{}.{}'.format(sys.version_info[0], sys.version_info[1]))\""
 
-            container_obj: Container = docker_client.containers.run(image=image,
-                                                                    command=shlex.split(command),
-                                                                    detach=True)
+            container_obj: Container = docker_client.containers.run(
+                image=image,
+                command=shlex.split(command),
+                detach=True
+            )
+            containers.append(container_obj)
             # Wait for container to finish
             container_obj.wait(condition="exited")
             # Get python version
             py_num = container_obj.logs()
             if isinstance(py_num, bytes):
                 py_num = float(py_num)
-                break
             else:
                 raise docker.errors.ContainerError
+            for _ in range(2):
```

Comment:
why did you move it here?

Reply:
because before that the code tried to use container_obj which can be not assigned